journal/original/07-chapter-6.adoc

```diff
@@ -36,6 +37,7 @@ for their minister was much set against the Quakers,
 and himself was an elder of their church.
 After some discourse of religion, we parted lovingly; I travelled on,
 and that night lay at an ale-house.
+//I didn't do much with this paragraph in the original? Is that right?
```

Comment:
Yup, perfect. I think I prefer to keep the original punctuation here for the most part because it's the original. 👍